### ISSUE TITLE
Support for removing a user's team assignments

### ISSUE THREAD

**Issue by Gby56** [veracode/veracode-api-py#45]
I think this comes down to the incremental vs partial choice in the api
I managed to add new teams to a users, it was really easy because I just had to give the new team's id and it was added on the existing... but I knew this would probably be a problem if I wanted to remove that after.

**Comment by Gby56** [comment]
Ah nevermind ! I could use the update_team and do the whole thing there also

**Comment by tjarrettveracode** [comment]
Yeah, the way you have to do it is to fetch the team's current info, remove the user from the list of team members, and update the team with the updated list. I don't think you need to specify incremental in that case…

**Comment by Gby56** [comment]
But I think the incremental is hardcoded so I can't do a full replacement of the team members

**Comment by Gby56** [comment]
@tjarrettveracode Here's why I think it's currently not possible to remove a user from a team, from the user's API, it's hardcoded incremental. So I'll implement the other way as you mentioned, take the whole team members and remove, and push that to the team again
https://github.com/veracode/veracode-api-py/blob/main/veracode_api_py/identity.py#L110

**Comment by Gby56** [comment]
I actually don't think there is an API in the client to get all the members in a team right now... so I can't do that pull, remove, replace either

**Comment by Gby56** [comment]
Created https://github.com/veracode/veracode-api-py/issues/46 because it's needed for this issue
